logger.go

```diff
@@ -18,12 +18,22 @@ func New(name string) *Logger {
 	}
 }
 
+// Checks whether this logger is enabled, sets self.IsEnabled, and returns the value.
+// This is a little kludgy but was introduced to allow dynamic log-name/log-level
+// changes to be made by programs at runtime. IsEnabled property is retained for API
+// compatibility.
+func (l *Logger) amIEnabled() bool {
```

Comment:
this naming confuses me. how is it different with `IsEnabled` ? Why do we have both ?